workitem/expression_compiler.go

```diff
@@ -110,8 +110,8 @@ func (c *expressionCompiler) getFieldName(fieldName string) (mappedFieldName str
 	if isColumnField {
 		return Column(WorkItemStorage{}.TableName(), mappedFieldName), false
 	}
-
-	if strings.Contains(fieldName, ".") {
+	// Check if the field name contains an underscore
+	if strings.Contains(fieldName, "_") {
```

Comment:
This reminds me that this piece of code needs some special treatment as the logic was previously to only handle `system.XXX` fields here and to distinguish between those fields that map to columns inside the `work_items` table and those that actually go to the `fields` column.

Reply:
@kwk I guess you're talking about this piece of code
https://github.com/fabric8-services/fabric8-wit/blob/bbd8f88c789943293680644ae8a49b5342a55575/workitem/expression_compiler.go#L108-L117

Do you think I should fix something? (I couldn't understand what did you want me to change)

Reply:
No No, this was just a mental note to myself.